layouts/MainLayout.tsx

```diff
@@ -1,17 +1,29 @@
 import Head from "next/head";
-import React, { ReactNode } from "react";
+import React from "react";
 
-type LayoutProps = {
-  children: ReactNode;
+type Props = {
+  title?: string | null;
+  children: React.ReactNode;
 };
 
-const MainLayout = ({ children }: LayoutProps): JSX.Element => (
+const MainLayout = ({ title, children }: Props): JSX.Element => (
   <>
     <Head>
-      <title>Tenancy & Household Check</title>
+      <title>Tenancy & Household Check{title ? ` - ${title}` : ""}</title>
+      <link
+        href="https://fonts.googleapis.com/css?family=Montserrat:200,300,400,500,700&display=swap"
+        rel="stylesheet"
+      />
```

Comment:
We have something to track that we need to fix this, right? Maybe add an issue to `lbh-frontend-react` or `lbh-frontend` (or both?).